src/data/matrix/basic.lean

```diff
@@ -1153,6 +1205,10 @@ by ext i j; refl
 lemma transpose_map {f : α → β} {M : matrix m n α} : Mᵀ.map f = (M.map f)ᵀ :=
 by { ext, refl }
 
+lemma transpose_col_eq_row (A : matrix m n α) (i : m) : (λ j, Aᵀ j i) = A i := rfl
+
+lemma transpose_row_eq_col (A : matrix m n α) (j : n) : Aᵀ j = (λ i, A i j):= rfl
+
```

Comment:
These lemmas seem bizarre to me - in the rare situation you actually need them, you could presumably just use `delta matrix.transpose`. Was there a reason you wanted them?

Reply:
I am not sure what is `delta matrix.transpose`. Could you explain? I wrote them as I do have used them in my own work, as such I think adding them is perhaps not a bad idea.

Reply:
By the way, why did the checks fail? @eric-wieser

Reply:
`delta matrix.transpose` is a tactic that means "replace `ᵀ` with it's definition"

Reply:
If we have `transpose_col_eq_row`, we can use it in rw or simp. I think I prefer to keep them.